docs/docs/native-docker-private-registry.md

```diff
@@ -4,34 +4,176 @@ title: Using a Private Docker Registry
 
 # Using a Private Docker Registry
 
-This document describes how to initiate a [Docker](https://docker.com/) pull from
-an authenticated private registry.
+As of Marathon 1.5, you can upload your private Docker registry credentials to a secret store, then reference them in your app or pod definition. This functionality is only available if you are using the Mesos containerizer. If you are using the Docker containerizer, follow [these instructions](#docker-containerizer) to use a private Docker registry.
 
-## Registry  1.0 - Docker pre 1.6 
+## Step 1: Create a Credentials File
+
+1. Log in to your private registry manually. This will create a `.docker` folder and a `.docker/config.json` file.
```

Comment:
I think it makes sense to not omit `~` in paths related to `config.json`. So, `a .docker folder and a .docker/config.json file` can be substituted with just `a ~/.docker/config.json` file. Also, in the world of filesystems, folders are called directories.